pages/account/troubleshooting/index.mdx

```diff
@@ -1,8 +1,59 @@
 ---
 meta:
-  title: Account Troubleshooting
-  description: Troubleshoot common issues with Scaleway accounts.
+  title: Account - Troubleshooting
+  description: Find troubleshooting resources for resolving issues with Account.
 content:
-  h1: Account Troubleshooting
-  paragraph: Troubleshoot common issues with Scaleway accounts.
+  paragraph: Find troubleshooting resources for resolving issues with Account.
+dates:
+  posted: 2023-10-05
+categories:
+  - account
 ---
+
+<ProductHeader
+  productName="Account troubleshooting"
+  productLogo="accountExperience"
+  description="Discover how to solve the most common issues in Scaleway account management. "
+/>
+
+## Featured Pages
+
+<Grid>
+    <DefaultCard
+      title="Account FAQ"
+      description="General info on Scaleway accounts."
+      url="/faq/account/"
+      label="See more"
+    />
+    <DefaultCard
+      title="How to manage your personal data"
+      description="Manage personal data under GDPR regulations in Scaleway."
```

Comment:
I found that long descriptions were cut off by the card, it only shows like the first 5 words